stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -726,6 +725,18 @@ public protocol _TensorArrayProtocolEnhanced : TensorArrayProtocol {
     where C.Element == CTensorHandle
 }
 
+
+extension _TensorArrayProtocolEnhanced {
+  func _dtypes() -> [TF_DataType] {
+    let count = _tensorHandleCount
```

Comment:
nit: consider removing the local var `count` as it's not doing much. 

Or I'd write:
```swift
    let count = Int(_tensorHandleCount)
```

and remove the cast in the next line.

Reply:
Done.